resources/vdb_template.yaml

```diff
@@ -1,18 +1,18 @@
 apiVersion: teiid.io/v1alpha1
 kind: VirtualDatabase
 metadata:
-  name: ${name}
+  name: $!DUMMY!$
 spec:
   replicas: 1
   env:
  
   build:
     source:
       ddl: |
-        CREATE DATABASE {name} OPTIONS (ANNOTATION 'provide your description here..');
-        USE DATABASE {name};
+        CREATE DATABASE $!DUMMY!$ OPTIONS (ANNOTATION 'provide your description here..');
```

Comment:
using a meaningful name,such as the one used for the variable in the code DDL_NAME_PLACEHOLDER,  instead of dummy would be easier to understand what is this dummy

Reply:
I have that addition in another PR for #112 already.